onnxscript/function_libs/torch_lib/graph_building.py

```diff
@@ -404,7 +404,7 @@ def add_initializer(self, name: str, value: torch.Tensor) -> TorchScriptTensor:
             # NOTE: Previously it raises when `name` is already set. This is relaxed
             # because this will be invoked multiple times when submodule is called
             # multiple times.
-            if name in self._initializers and self._initializers[name] != value:
```

Comment:
So this comparison was invalid right?

Reply:
It does tensor value comparison, i.e. torch.neq(a, b). It returns a bool tensor.

The check should instead see if they are same tensor object.